velox/functions/lib/string/StringImpl.h

```diff
@@ -147,6 +147,27 @@ FOLLY_ALWAYS_INLINE int32_t charToCodePoint(const T& inputString) {
   return codePoint;
 }
 
+/// Returns the sequence of Unicode code point of characters in the an input
+/// string.
+template <typename T>
+FOLLY_ALWAYS_INLINE std::vector<int32_t> stringToCodePoint(
```

Comment:
When inputString is all-ASCII, each byte represents one character and the codepoint of that character equals the integer value of the byte. Hence, an all-ASCII version of this function can be implemented much more efficiently by casting char* to int8_t*.